src/mock-data/quote.ts

```diff
@@ -37,4 +44,5 @@ export const MOCK_QUOTE: Record<string, Omit<QuoteRequestBody, 'address'>> = {
   quoteOutNigeriaCUSD,
   quoteOutXOFCUSD,
   quoteInXOFCUSD,
+  quoteOutBrazilCREAL: quoteBrazilCREAL,
```

Comment:
```suggestion
  quoteBrazilCREAL,
```
not important but a bit nicer

Reply:
ok, but will need to change quote_out_mock on env file.

Reply:
oh whoops I didn't read closely and didn't see the name difference

Reply:
lol